llvm/lib/Target/DirectX/DirectXTargetTransformInfo.cpp

```diff
@@ -54,6 +54,7 @@ bool DirectXTTIImpl::isTargetIntrinsicTriviallyScalarizable(
   case Intrinsic::dx_saturate:
   case Intrinsic::dx_splitdouble:
   case Intrinsic::dx_wave_readlane:
+  case Intrinsic::dx_wave_reduce_or:
```

Comment:
if you add a line here then you should be testing this intrinsic gets scalarized